src/main/resources/io/jenkins/plugins/designlibrary/EmptyStates/index.properties

```
@@ -0,0 +1,8 @@
+noticeTitle=No plugins installed
+parameters=Supported Parameters:
+parameters.title=The message displayed in the notice box
+parameters.icon=The icon shown with the message
+descriptionChildContent=You can also include child content within the notice for additional instructions.
+childSectionTitle=Child content inside notices
+childNoticeTitle=No jobs created
+newJobText=Create your first job and start building

```

I wouldn't make the label too long
```suggestion
newJobText=Create your first job

```